MMM-COVID19.js

```diff
@@ -232,6 +233,9 @@ Module.register("MMM-COVID19", {
 
         countryNameCell.innerHTML = this.translate(countryName)
         countryNameCell.className = this.config.infoRowClass
+	      if (countryName === this.config.highlightCountry) {
```

Comment:
sorry but you have here 6 extra whitespaces

Reply:
Darn, shoulda seen it myself. Fixed. Thanks for being patient with me!

Reply:
@hweigel sorry but I still see it wrong (indentation)

Reply:
@bibaldo I see it here in the change browser too, but when I open the change, it shows the correct indentation:

![Clipboard-3](https://user-images.githubusercontent.com/60696861/78114238-3d45df00-7401-11ea-98db-cf96eed827ef.jpg)

Reply:
don't worry, we will merge it anyway and I will fix it on master.

Sorry for the inconveniences